tests/integration/monitors/run.rs

```diff
@@ -1,47 +1,21 @@
-use crate::integration::{mock_endpoint, register_test, EndpointOptions};
+use crate::integration::register_test;
 
 #[test]
 fn command_monitors_run() {
-    let _server = mock_endpoint(
```

Comment:
Would be great if I could mock the envelopes sent. to validate it's sending the right things.

Doesn't look like we're doing anything like that anywhere yet

Reply:
You can with log level test.

Reply:
Ah nvm, envelopes do not implement this now, only events do 👀